Code/src/main/java/nl/utwente/viskell/haskell/expr/OtherMatchBinder.java

```diff
@@ -0,0 +1,15 @@
+package nl.utwente.viskell.haskell.expr;
+
+import nl.utwente.viskell.haskell.type.Type;
+
+public class OtherMatchBinder extends Binder {
```

Comment:
This could probably use some Javadoc (what does it represent, and why do we need it?).